srcpkgs/tpm2-tss/template

```diff
@@ -14,6 +14,8 @@ homepage="https://github.com/tpm2-software/tpm2-tss"
 distfiles="https://github.com/tpm2-software/tpm2-tss/archive/${version}.tar.gz"
 checksum=da818b8cb82630f561ed088ef3204b243a8f1de680623a09d775deb6014c4c99
 
+system_accounts="tss tss"
```

Comment:
Should it just be `system_accounts="tss"` ?

This is what is done in `srcpkgs/trousers/template`.  Most other packages have just one field specified.

Reply:
According to the Manual:
This specifies the names of the new system users/groups to be created, separated by blanks, i.e system_accounts="_foo _blah:22"

Reply:
I think you only need to specify it once.  It will create both a user and group with the name `tss`.